src/overlays/actors/ovl_En_GeldB/z_en_geldb.c

```diff
@@ -33,93 +97,1466 @@ const ActorInit En_GeldB_InitVars = {
+        func_80A391D8(this, globalCtx);
+    }
+    Collider_CylinderUpdate(&this->actor, &this->bodyCollider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->bodyCollider.base);
+    if ((this->actionState >= GELDB_READY) && (this->spinAttackState < 2) &&
+        ((this->actor.dmgEffectTimer == 0) || !(this->actor.dmgEffectParams & 0x4000))) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->bodyCollider.base);
+    }
+    if ((this->actionState == GELDB_BLOCK) && (this->skelAnime.animCurrentFrame == 0.0f)) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->blockCollider.base);
+    }
+    if (this->swordState > 0) {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->swordCollider.base);
+    }
+    if (this->blinkState == 0) {
+        if ((Math_Rand_ZeroOne() < 0.1f) && ((globalCtx->gameplayFrames & 3) == 0)) {
+            this->blinkState++;
+        }
+    } else {
+        this->blinkState = (this->blinkState + 1) & 3;
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_GeldB/func_80A38054.s")
+s32 EnGeldB_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                             void* thisx) {
+    EnGeldB* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_GeldB/func_80A380EC.s")
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_geldB.c", 2507);
+    if (limbIndex == 3) {
+        rot->z += this->headRot.x;
+        rot->x += this->headRot.y;
+        rot->y += this->headRot.z;
+    } else if (limbIndex == 6) {
+        gDPPipeSync(POLY_OPA_DISP++);
+        gDPSetEnvColor(POLY_OPA_DISP++, 80, 60, 10, 255);
+    } else if ((limbIndex == 11) || (limbIndex == 16)) {
+        gDPPipeSync(POLY_OPA_DISP++);
+        gDPSetEnvColor(POLY_OPA_DISP++, 140, 170, 230, 255);
+        gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 255, 255, 255, 255);
+    } else {
+        gDPPipeSync(POLY_OPA_DISP++);
+        gDPSetEnvColor(POLY_OPA_DISP++, 140, 0, 0, 255);
+    }
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_geldB.c", 2529);
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_GeldB/func_80A38290.s")
+void EnGeldB_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
+    static Vec3f footOffset = { 300.0f, 0.0f, 0.0f };
+    static Vec3f swordTipOffset = { 0.0f, -3000.0f, 0.0f };
+    static Vec3f swordHiltOffset = { 400.0f, 0.0f, 0.0f };
+    static Vec3f swordQuadOffset1 = { 1600.0f, -4000.0f, 0.0f };
+    static Vec3f swordQuadOffset0 = { -3000.0f, -2000.0f, 1300.0f };
+    static Vec3f swordQuadOffset3 = { -3000.0f, -2000.0f, -1300.0f };
+    static Vec3f swordQuadOffset2 = { 1000.0f, 1000.0f, 0.0f };
+    static Vec3f zeroVec = { 0.0f, 0.0f, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_GeldB/func_80A38348.s")
+    Vec3f swordTip;
```

Comment:
dont need a break between static and normal variables, can treat them all as the same list